projects/ngneat/transloco/src/lib/tests/transloco.directive.spec.ts

```diff
@@ -141,6 +147,18 @@ describe('TranslocoDirective', () => {
       testScopedTranslation(host);
     }));
 
+    // TODO test loading of scoped translations with specified in component alias
```

Comment:
?

Reply:
Yeah, I need help with writing this test :) 
As I mentioned in the PR's 'Other Info' section, I wasn't able to write the test. 
How can I override component's providers in Spectator? sending it to createHost didn't work for me. Is it because of the Spectator version used for Transloco or am I missing something?

Reply:
Yes, we need to update Spectator. For now, you can create a new describe using a new service.

Reply:
Okay, thanks!

Reply:
I've pushed all requested changes. However, I am still not sure about this test: if I update scopeMapping using service, it will not update the actual translation because it's just updating the object, no event emission here. So scope alias will not take affect. Or I don't get what you mean? Sorry if I miss something easy, quite  new to Angular world :)

Reply:
I will do it. Can you please merge `feature/t-function` into your branch? I want to release v2 with this feature.